test/app-luatest/gh_7288_console_flavours_vs_txn_test.lua

```diff
@@ -0,0 +1,171 @@
+local server = require('test.luatest_helpers.server')
+local console = require('console')
+local fiber = require('fiber')
+local fio = require('fio')
+local string = require('string')
+local t = require('luatest')
+
+local alocal = {
+    start = function()
+        box.cfg{memtx_use_mvcc_engine = true}
```

Comment:
Let's set `work_dir` to a location inside `VARDIR` so as not to pollute the current directory if the test is run via `luatest`. Also, let's create the test Unix socket (`.gh-7288.sock`) in `VARDIR` as well for the same reason.